mtags/src/main/scala/scala/meta/internal/metals/Docstrings.scala

```diff
@@ -54,14 +66,17 @@ class Docstrings(index: GlobalSymbolIndex) {
     }
   }
 
-  private def indexSymbolDefinition(defn: SymbolDefinition): Unit = {
+  private def indexSymbolDefinition(
+      defn: SymbolDefinition,
+      occFn: Option[SymbolOccurrence => Unit]
+  ): Unit = {
     defn.path.toLanguage match {
       case Language.JAVA =>
         JavadocIndexer
           .foreach(defn.path.toInput)(cacheSymbol)
```

Comment:
Though we won't expire the cache for Javadoc symbols, it might be better to pass `occFn` function to `JavadocIndexer` for consistency?